scripts/script_utils.py

```diff
@@ -146,10 +129,79 @@ def group_norm_symbolic(g, input_blob, num_groups, weight, bias, eps, cudnn_enab
 
     return output
 
+
 def is_config_parameter_set_from_command_line(cmd_line_opts, parameter_name):
     # Note that cmd_line_opts here should be compatible with
     # the function yacs.config.CfgNode.merge_from_list
     if not cmd_line_opts:
         return False
     key_names = cmd_line_opts[0::2]
     return (parameter_name in key_names)
+
+
+def put_on_the_device(model, use_gpu=True, gpu_num=1, num_aux_models=0, split_models=False):
```

Comment:
why? In this function, the extra devices for aux models are initialized too

Reply:
It may additionally return indexes of devices for aux models if some particular non-default values of  kwargs are provided, but certainly puts the passed main model to some device.